lib/tasks/mastodon.rake

```diff
@@ -33,6 +33,36 @@ namespace :mastodon do
     end
   end
 
+  desc 'Add a user by providing their email, username and initial password.' \
+       'The user will receive a confirmation email, then must reset their password before logging in.'
+  task add_user: :environment do
+    print 'Enter email: '
+    email = STDIN.gets.chomp
+
+    print 'Enter username: '
+    username = STDIN.gets.chomp
+
+    print 'Create user and send them confirmation mail (y/n): '
+    confirm = STDIN.gets.chomp
+    puts
+
+    if confirm == 'y'
```

Comment:
@Gargron, As stated in this answer '[Y/n]' mean 'y' is default option, ie, if no character is entered and enter key is pressed, 'y' would be selected. Do you want me to change the current behavior (of 'no' being the default option) and make 'yes' the default action?